sql/updates/world/3.3.5/2016_05_12_00_world_PROCS.sql

```diff
@@ -0,0 +1,10 @@
+-- Blazing Speed
+DELETE FROM spell_ranks where first_spell_id = 31641;
+INSERT INTO spell_ranks VALUES ( 31641,31641,1),( 31641,31642,2);
+DELETE FROM spell_script_names where ScriptName = 'spell_mage_blazing_speed';
+INSERT INTO spell_script_names VALUES (-31641,'spell_mage_blazing_speed');
+-- Blessed Recovery
```

Comment:
Preliminary update; apply TC codestyle to your SQL queries:
```sql
-- Mage spell 31641, Blazing Speed
DELETE FROM `spell_ranks` WHERE `first_spell_id`= 31641;
INSERT INTO `spell_ranks` (`first_spell_id`,`spell_id`,`rank`) VALUES
(31641, 31641, 1),
(31641, 31642, 2);

DELETE FROM `spell_script_names` WHERE `ScriptName`= 'spell_mage_blazing_speed';
INSERT INTO `spell_script_names` (`spell_id`,`ScriptName`) VALUES
(-31641, 'spell_mage_blazing_speed');

-- Priest spell 27811, Blessed Recovery
DELETE FROM `spell_ranks` WHERE `first_spell_id`= 27811;
INSERT INTO `spell_ranks` (`first_spell_id`,`spell_id`,`rank`) VALUES
(27811, 27811, 1),
(27811, 27815, 2),
(27811, 27816, 3);

DELETE FROM `spell_script_names` WHERE `ScriptName`= 'spell_priest_blessed_recovery';
INSERT INTO `spell_script_names` (`spell_id`,`ScriptName`) VALUES
(-27811, 'spell_priest_blessed_recovery');
```
These 8 queries can also be merged into 4, but let's keep them divided until the PR is ready for merging.